.github/workflows/todo_check.yml

```diff
@@ -0,0 +1,28 @@
+name: TODO checks
```

Comment:
On my gihub interface I see below this line a check failure from "GitHub Actions / New TODOs" ,,, Am I correct that it is the todo_check that applies to itself and does not pass ?

Reply:
Yes, this check is rather simple and just looks for modified lines that contain "TODO" or "FIXME", so gets triggered by itself. This only happens once when adding the file, so shouldn't be an issue in future PRs.